include/dlaf/factorization/cholesky/mc/cholesky_L.h

```diff
@@ -144,10 +147,13 @@ void cholesky_L(comm::CommunicatorGrid grid, Matrix<T, Device::CPU>& mat_a) {
         // Avoid useless communication if one-column communicator and if on the last column
         if (col_comm_size > 1 && k != (mat_a.nrTiles().cols() - 1)) {
           // Broadcast the panel column-wise
-          hpx::dataflow(hpx::util::unwrapping([](auto&& tile, auto&& comm_wrapper) {
```

Comment:
Here `executor_mpi` is missing.
It should be fixed for the comparison benchmarks.

Reply:
PR: #246